src/app/pages/MediaAssetPage/MediaAssetPage.jsx

```diff
@@ -187,10 +185,6 @@ const MediaAssetPage = ({ pageData }) => {
     }
   `;
 
-  const MostWatchedWrapper = styled.div`
-    padding-bottom: ${GEL_SPACING_QUAD};
-  `;
```

Comment:
This is likely to be the reason for the Chromatic diff - padding at the bottom that is no longer required 🙌

Reply:
excellent, that's fine then :)